templates/taxi/car_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete car {{ car.model }}</h1>
+  <br>
+  <h5>Are you sure you want to delete this car?</h5>
+
+  <form action="" method="post">
+  {% csrf_token %}
+    <input type="submit" value="yes" class="btn btn-danger">
+  </form>
```

Comment:
Can you add a "No" or "Cancel" button? This is not a mistake, but it looks nicer, and more complete